### DISCUSSION TITLE
Google Play removed my application from playstore by mentioning deprecated SDK issue.

### DISCUSSION THREAD

**Discussion by wasiquehaider** [invertase/react-native-firebase#5755] [General]
A couple of months ago when I tried to upload the version of my app on playstore google play rejected that mentioning they  have detected that my app includes the deprecated Personalised Ad Consent SDK, which is not approved for use in child-directed services. Now google play completely removed the app from playstore mentioning the same issue. Please can I get any response from the lead moderators. The Pictures is also attached below.

<img width="636" alt="Screenshot 2021-09-30 at 1 09 29 PM" src="https://user-images.githubusercontent.com/26715284/135413381-c46c03da-fa19-493c-a841-3b988e6badeb.png">

**Comment by mikehardy** [comment]
Hi there! As far as I know there are no problems with the current versions of the firebase-android-sdk if used correctly.
You have not provided any information about your application, the versions in use, the packages in use etc that would help us diagnose your issue in any way, so no further action on our part is possible.

**Reply by wasiquehaider** [reply]
here you go

package.json

{
  "name": "abcApp",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "android": "npx react-native run-android",
    "apk:test:release": "cd android; ./gradlew clean; ./gradlew assembleRelease; cd ..;",
    "apk:test:debug": "npx react-native run-android --variant=release",
    "ios": "npx react-native run-ios",
    "start": "npx react-native start",
    "reconnect": "adb reverse tcp:8081 tcp:8081",
    "gradlewClean": "cd android; ./gradlew clean; cd ..;",
    "podInstall": "cd ios; pod install; cd ..;",
    "resetPackager": "npx react-native start --reset-cache",
    "test": "jest",
    "lint": "eslint ."
  },
  "dependencies": {
    "@react-native-community/async-storage": "^1.10.0",
    "@react-native-community/audio-toolkit": "^2.0.3",
    "@react-native-community/masked-view": "^0.1.10",
    "@react-native-community/netinfo": "^5.8.1",
    "@react-native-community/push-notification-ios": "^1.8.0",
    "@react-native-community/viewpager": "^3.3.0",
    "@react-native-firebase/app": "^11.4.1",
    "@react-native-firebase/crashlytics": "^11.4.1",
    "@react-native-firebase/messaging": "^11.4.1",
    "@react-navigation/native": "^5.1.6",
    "@react-navigation/stack": "^5.2.13",
    "@sentry/react-native": "^2.2.1",
    "apisauce": "^1.1.1",
    "async": "^3.2.0",
    "lodash": "^4.17.15",
    "moment": "^2.24.0",
    "react": "16.13.1",
    "react-native": "0.63.3",
    "react-native-admob": "^2.0.0-beta.6",
    "react-native-animatable": "^1.3.3",
    "react-native-bootsplash": "^2.2.4",
    "react-native-device-info": "^7.0.2",
    "react-native-fs": "^2.16.6",
    "react-native-gesture-handler": "^1.6.1",
    "react-native-iap": "^5.2.14",
    "react-native-inappbrowser-reborn": "^3.5.1",
    "react-native-indicators": "^0.17.0",
    "react-native-keyboard-manager": "^4.0.13-16",
    "react-native-modal": "^11.5.6",
    "react-native-offline": "^5.6.1",
    "react-native-orientation-locker": "^1.1.8",
    "react-native-popup-dialog": "^0.18.3",
    "react-native-purchases": "^3.2.0",
    "react-native-push-notification": "^7.0.0",
    "react-native-reanimated": "^1.8.0",
    "react-native-safe-area-context": "^0.7.3",
    "react-native-screens": "^2.5.0",
    "react-native-share": "^3.3.2",
    "react-native-snap-carousel": "^3.9.0",
    "react-native-system-setting": "^1.7.6",
    "react-native-webview": "^10.10.0",
    "react-redux": "^7.2.0",
    "redux": "^4.0.5",
    "redux-logger": "^3.0.6",
    "redux-persist": "^6.0.0",
    "redux-saga": "^1.1.3"
  },
  "devDependencies": {
    "@babel/core": "^7.8.4",
    "@babel/runtime": "^7.8.4",
    "@react-native-community/eslint-config": "^1.1.0",
    "babel-jest": "^25.1.0",
    "eslint": "^6.5.1",
    "jest": "^25.1.0",
    "metro-react-native-babel-preset": "^0.59.0",
    "react-test-renderer": "16.13.1"
  },
  "jest": {
    "preset": "react-native"
  }
}
